e2e/scripts/run_android_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Start emulator and run tests
+emulator @Pixel_8.1 & ./node_modules/.bin/wdio test/configs/android.conf.js
```

Comment:
can we make sure in the follow up ticket we grab the first available emulator or even pass an environment variable to select the emulator? I'm think of running the tests in different versions of Android

Reply:
For reference, just created https://mattermost.atlassian.net/browse/MM-16300